docs/troubleshooting.rst

```diff
@@ -163,33 +163,29 @@ By default, Open edX comes with a `limited set <https://github.com/openedx/edx-p
 When I make changes to a course in the CMS, they are not taken into account by the LMS
 --------------------------------------------------------------------------------------
 
-This issue should only happen in development mode. Long story short, it can be solved by creating a Waffle switch with the following command::
+This issue should only happen in development mode. Long story short, it is solved by creating a Waffle switch with the following command::
 
     tutor dev run lms ./manage.py lms waffle_switch block_structure.invalidate_cache_on_publish on --create
 
-If you'd like to learn more, please take a look at `this Github issue <https://github.com/overhangio/tutor/issues/302>`__.
+To learn more, check out `this GitHub issue <https://github.com/overhangio/tutor/issues/302>`__.
 
-High resource consumption on ``tutor images build`` by docker 
--------------------------------------------------------------
+High resource consumption on ``tutor images build``
+---------------------------------------------------
 
-This issue can occur when building multiple images simultaneously by Docker, issue specifically related to BuildKit.
-
-
-Create a buildkit.toml configuration file with the following contents::
+Some Docker images include many independent layers which are built in parallel by BuildKit. As a consequence, building these images will use up a lot of resources, sometimes even crashing the Docker daemon. To bypass this issue, we should explicitely limit the `maximum parallelism of BuildKit <https://docs.docker.com/build/buildkit/configure/#max-parallelism>`__. Create a ``buildkit.toml`` configuration file with the following contents::
 
     [worker.oci]
     max-parallelism = 2
 
-This configuration file limits the number of layers built concurrently to 2, which can significantly reduce resource consumption.
-
-Create a builder that uses this configuration::
+This configuration file limits the number of layers built concurrently to 2, but we should select a value that is appropriate for our machine.
 
-    docker buildx create --use --name=<name> --driver=docker-container --config=/path/to/buildkit.toml
+Then, create a builder named "max2cpu" that uses this configuration, and start using it right away::
 
-Replace <name> with a suitable name for your builder, and ensure that you specify the correct path to the buildkit.toml configuration file.
+    # don't forget to specify the correct path to the buildkit.toml configuration file
+    docker buildx create --use --name=max2cpu --driver=docker-container --config=/path/to/buildkit.toml
 
 Now build again::
 
-    tutor images build
+    tutor images build all
 
-All build commands should now make use of the newly configured builder. To later revert to the default builder, run ``docker buildx use default``. 
+All build commands should now make use of the newly configured builder. To later revert to the default builder, run ``docker buildx use default``.
```

Comment:
As mentioned on https://github.com/overhangio/tutor/pull/1005#issuecomment-2057053474, to switch back to default builder, `docker context use default` should be used.

Reply:
Actually, we resolved to keep recommending the `buildx` command.